tests/test_script.py

```diff
@@ -117,7 +117,7 @@ async def test_script_evaluateThrowingError_exceptionReturned(websocket,
 
 @pytest.mark.asyncio
 async def test_script_evaluateDontWaitPromise_promiseReturned(websocket,
-      context_id):
+                                                              context_id):
```

Comment:
Can we move reformat to another PR? It's a subject to discuss, as our auto-formats are different

Reply:
Oh I'm sorry I only saw this after I merged the commit. I had to run `npm run prettier` because of a merge conflict with Alex's changes in a previous commit.

There's definitely some improvements we can do to improve the formatting (especially w.r.t. reducing noise like in this case), maybe by integrating it with the CI.